addons/hr_holidays/migrations/11.0.1.5/pre-migration.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+field_renames = [
+    ('hr.holidays', 'hr_holidays', 'manager_id2', 'second_approver_id')
+]
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    openupgrade.rename_fields(env, field_renames)
+    openupgrade.logged_query(
```

Comment:
You should use for better completion the new openupgradelib method `add_fields`: https://github.com/OCA/openupgradelib/blob/7d9650376e1b64838b6caf2e90581176ac683808/openupgradelib/openupgrade.py#L1815

I'm adding it already

Reply:
Perfect @pedrobaeza 

I didn't know this method.

Reply:
It was added recently by me as I observed that there was a gap (the created XML-ID) and to try to simplify which PG type to use for adding the column.

On v12 won't be needed for the XML-ID part.